### PR TITLE
Add cassy monitoring

### PR DESCRIPTION
**Pull request by tei-k**
# Description

https://scalar-labs.atlassian.net/browse/DLT-6050

### CODE REVIEW

**Comment by tei-k** [comment]
Thanks!

I moved the rule to `general_alert.rules.yml` same as reaper and fixed yml format. 🙏

**Comment by feeblefakie** [comment]
@ymorimo Please take a look !

File: modules/universal/monitor/provision/prometheus/general_alert.rules.yml [hunk 1]
```diff
@@ -1,82 +1,89 @@
...
-    annotations:
-      summary: "Reaper container is down"
-      description: "The reaper container on {{ $labels.instance }} of role {{ $labels.role }} down for more than 15 seconds"
+      - alert: CassyContainerDown
+        expr: time() - container_last_seen{container_label_com_docker_compose_service="cassy"} > 15
+        labels:
+          severity: warning
+        annotations:
+          summary: "Cassy container is down"
+          description: "The cassy container on {{ $labels.instance }} of role {{ $labels.role }} down for more than 15 seconds"
```
Comment (ymorimo) [inline modules/universal/monitor/provision/prometheus/general_alert.rules.yml:57]: These lines are the actual changes, right? (It would be helpful if you could create a separate PR for formatting only.)

**Comment by feeblefakie** [comment]
@tei-k @ymorimo From next time, let's try to refactor things in another PR as much as possible so that code review will be much easier. 🙇‍♂️
It is also noted here. https://github.com/scalar-labs/team-handbook/blob/master/engineering/guidelines/commit.md#pr-policy

File: modules/universal/monitor/provision/prometheus/general_alert.rules.yml (hunk 1 above)
Comment (tei-k) [reply modules/universal/monitor/provision/prometheus/general_alert.rules.yml:57]: @ymorimo 

Thanks! Also changed the following file.
`modules/universal/monitor/provision/prometheus/prometheus.yml.j2` 
```
 -   {% for s in ['scalardl-blue','scalardl-green','monitor','reaper','envoy']|list %}
 +   {% for s in ['scalardl-blue','scalardl-green','monitor','reaper','cassy','envoy']|list %}
```

**Comment by tei-k** [comment]
@feeblefakie 

>From next time, let's try to refactor things in another PR as much as possible so that code review will be much easier

Got it.
But I think it is fine that fix the code formatting together. 🙄
